sdk/communication/azure-communication-callingserver/src/main/java/com/azure/communication/callingserver/models/events/SubscriptionValidationEvent.java

```diff
@@ -0,0 +1,65 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.communication.callingserver.models.events;
+
+import com.azure.core.annotation.Immutable;
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/** The SubscriptionValidationEvent model. */
+@Immutable
+public final class SubscriptionValidationEvent implements CallingServerBaseEvent {
```

Comment:
This event is from event grid and not related to call automation exclusively - should we include them in our SDK?

Reply:
Same as the IncomingEvent. Should we discuss this on Standup?

Reply:
This is technically part of EventGrid, and handled by their SDK (just like incoming call) - So we would implement something that has been already implemented somewhere (or will be implemented)
Yes, we can bring up the discussion during the standup for further discussion

Reply:
fixed